cumulus/polkadot-parachain/src/chain_spec/people.rs

```diff
@@ -82,6 +96,10 @@ impl PeopleRuntimeType {
 				"westend-development",
 				ParaId::new(1004),
 			))),
+			other => Err(std::format!(
+				"No default config present for {:?}, you should provide a chain-spec as json file!",
```

Comment:
This would still give an error if the id in the json starts with `people-`, unless the fallthrough case for `from_str` is changed to return the `Runtime::default()`. Otherwise it will fail with e.g. `"Value 'people-kusama-local' is not configured yet"`

Reply:
yes, exactly, `load_config` fails for `people-kusama-local`, it means that `polkadot-parachain` does not provide default configuration for that, instead, you have to generate chain-spec from fellows repo with `chain-spec-builder people-kusama-local > chain-spec-people-kusama-local.json` and then run `./polkadot-parachain --chain chain-spec-people-kusama-local.json` and then you don't end up in this `other` branch

Reply:
I mean because of [this line](https://github.com/paritytech/polkadot-sdk/blob/bko-coretime-people/cumulus/polkadot-parachain/src/chain_spec/people.rs#L53), even if you specify a correct json, if the id in it is `people-*` and not included in the list in from_str, it will fail with the value not configured error.

`people-kusama-local` was a very bad example, since it's included there 🤦‍♂️

Reply:
I think I know what you mean, e.g. if you use `coretime-westend-my-super-duper-spec` it will fail on that `from_str` even if chain spec is ok.
Yes, let's improve (or remove this chain specific stuff code and rely just on the chain spec) in the https://github.com/paritytech/polkadot-sdk/issues/3944